eng/common/scripts/git-branch-push.ps1

```diff
@@ -50,18 +51,32 @@ if (!(git remote | ? {$_ -eq $RemoteName}))
         Write-Error "Unable to add remote LASTEXITCODE=$($LASTEXITCODE), see command output above."
         exit $LASTEXITCODE
     }
+    Write-Host "git fetch $RemoteName"
```

Comment:
I don't understand why this is needed? Also I don't think this script should be in the business of doing clone directly like you are doing below. Can you help me understand the scenario you are after?